sys/usb/usbus/cdc/ecm/cdc_ecm.c

```diff
@@ -0,0 +1,376 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup usbus_cdc_ecm
+ * @{
+ * @file USBUS implementation for ethernet control model
+ *
+ * @author  Koen Zandberg <koen@bergzand.net>
+ * @}
+ */
+
+#include "msg.h"
+#include "tsrb.h"
+
+#include "event.h"
+#include "kernel_defines.h"
+#include "luid.h"
+#include "net/ethernet.h"
+#include "net/eui48.h"
+#include "fmt.h"
+#include "usb/descriptor.h"
+#include "usb/cdc.h"
+#include "usb/usbus.h"
+#include "usb/usbus/control.h"
+#include "usb/usbus/cdc/ecm.h"
```

Comment:
Are they all needed ?

Reply:
No, removed `msg.h`, the others are all required I think. Also checked `cdc_ecm_netdev.h` for includes